tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Utility/ConfigSettings/GeneralSettings.cs

```diff
@@ -36,6 +36,7 @@ public class GeneralSettings
         public string KeyVaultURI { get; set; }
         public string GraphUrl { get; set; }
         public bool IsBackwardCompatible { get; set; }
+        public bool IsClientMappedWithHierachy { get; set; }

```

Comment:
Added one more property in the general section of appsettings.json called "IsClientMappedWithHierachy". This will decide whether the taxonomy hierarchy needs to be filtered for client selectin or not